templates/pytorch/nightly/fs-transformer.libsonnet

```diff
@@ -52,76 +87,47 @@ local utils = import "../../utils.libsonnet";
       },
     },
   },
+  local checkpoint_local = base.Functional {
+    modelName: "fs-checkpoint-local",
+    command: utils.scriptCommand(
+      |||
+        %(common)s  --max-epoch=1 \
+          --save-dir=/tmp/checkpoints
+        %(common)s  --max-epoch=2 \
```

Comment:
the goal here is to test loading/resuming from the chpt saved by the previous command.